client/src/components/content/Content.js

```diff
@@ -1,37 +1,21 @@
 import React, {Component} from 'react'
+import _ from 'lodash'
 import TopPanel from '../topPanel/TopPanel'
 import MessageList from '../message/MessageList'
-import {Loader} from '../ui'
-import {getReportsByTag} from '../../utils/serverApi'
 import './Content.scss'
 
 class Content extends Component {
-  state = {
-    loading: true,
-  }
 
-  async componentDidMount() {
-    const response = await getReportsByTag(this.props.tag)
-    this.setState({...response, loading: false})
+  shouldComponentUpdate(nextProps) {
+    return !_.isEqual(this.props.reportData.reports, nextProps.reportData.reports)
```

Comment:
Can there be a possibility that reports stay the same but tag changes? :slightly_smiling_face: ..I think it would be more correct to add check for `tag` too.